public/css/genericStyles.css

```diff
@@ -2,6 +2,10 @@
     box-sizing: border-box;
 }
 
+.bold {
```

Comment:
not good naming - its only used on a couple of hiring partner names so make the class more semantically named